### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by bsipocz**

### CODE REVIEW

File: astropy_helpers/commands/build_sphinx.py [hunk 1]
```diff
@@ -202,7 +202,7 @@ def run(self):
                     webbrowser.open(fileurl)
                 else:
                     log.warn('open-docs-in-browser option was given, but '
-                             'the builder is not html! Ignogring.')
+                             'the builder is not html! Ignoring.')
```
Comment (embray) [inline astropy_helpers/commands/build_sphinx.py]: Aww, but Ignogring sounds so...Tolkeinesque.

Comment (bsipocz) [inline astropy_helpers/commands/build_sphinx.py]: Oops, I didn't notice it was intentional :wink:

Comment (astrofrog) [inline astropy_helpers/commands/build_sphinx.py]: hf2cm

Comment (embray) [inline astropy_helpers/commands/build_sphinx.py]: You made my day.

Comment (pllim) [inline astropy_helpers/commands/build_sphinx.py]: :+1: to adding Ned Stark into our error messages.

Comment (embray) [inline astropy_helpers/commands/build_sphinx.py]: hf4cb

Comment (pllim) [inline astropy_helpers/commands/build_sphinx.py]: :+1: :+1: :+1:

**Pull request merged**